src/FastRandom.cpp

```diff
@@ -8,17 +8,19 @@
 
 #include <random>
 
-#ifdef _WIN32
-	#define thread_local static __declspec(thread)
-#elif defined __APPLE__
-	#define thread_local static __thread
+#if defined (__GNUC__)
+	#define ATTRIBUTE_TLS static __thread
```

Comment:
@tigerw Yes, it's available since GCC 4.8. The point is Apple's clang does not support it. And that clang is the default in OSX when you get Xcode tools.